euphonic/util.py

```diff
@@ -444,8 +444,8 @@ def _cell_vectors_to_volume(cell_vectors: Quantity) -> Quantity:
 
 
 def _get_unique_elems_and_idx(
-        all_elems: Sequence[Tuple[Union[int, str], ...]]
-        ) -> 'OrderedDict[Tuple[Union[int, str], ...], np.ndarray]':
+        all_elems: Sequence[tuple[int | str, ...]]
+        ) -> 'OrderedDict[tuple[int | str, ...], np.ndarray]':
```

Comment:
Does ordered dict need to be here if post 3.7?

Reply:
OrderedDict is not deprecated and can still be useful for expressing that the order-conservation is important to an algorithm.

In this case it's not _very_ important, though, so can be cleaned up.